lib/dradis/plugins/csv/importer.rb

```diff
@@ -0,0 +1,19 @@
+module Dradis::Plugins::CSV
+  class Importer < Dradis::Plugins::Upload::Importer
+    def self.templates
+      {}
+    end
+
+    def import(params={})
+      logger.info { 'Uploading CSV file...' }
+
+      uid = @logger.uid
+
+      # SEE: app/controllers/dradis/plugins/csv/upload_controller.rb
+      Resque.redis.set(uid, File.basename(params[:file]))
```

Comment:
don't we need a better Redis key naming convention?

Reply:
Added a redis prefix for the ID: `addons-csv-upload-<job_id>`